readme.md

```diff
@@ -123,6 +140,13 @@ Your Answer:
 
 Describe some of the common git workflows for teams (fork and pull request, etc).
 
-```text
+```
+git merge
+git rebase
+git pull
+git push
+git clone
+git stash
+
 
 ```
```

Comment:
While these commands will be used, we're looking for higher-level approaches that can be used. There are a number of answers, some of which are included below...

```
There are plenty of workflows a team can use:

1. Everyone is a contributor, and simply pushes their work to master.
2. Everyone is a contributor, but pushes their work to feature branches. Other team members merge in the branches at their discretion. Pull Requests may or may not be used.
3. Everyone forks the repo, and creates Pull Requests from their fork to the master. One or more people are in charge of reviewing and approving Pull Requests.

In addition to the above, teams may set their own policies on code review, when PRs can be merged, etc.

Additionally, teams may decide on a convention for naming their branches, and how changes get merged into the master branch (e.g. there may be an intermediate Development branch for integrating multiple PRs and testing how they work together.) One such example is the git flow model.
```